templates/registration/totp_enable.html

```diff
@@ -41,6 +41,24 @@
     </style>
 {% endblock %}
 
+{% block js_media %}
+    <script>
+        $(function () {
+            if (window.navigator.userAgent.indexOf('Edge') >= 0) {
```

Comment:
Doesn't our caniuse app support this kind of lookup server-side?

Reply:
Yes, it does. But since all the compat work is done in JS, might as well detect it in the JS instead of passing it along.

Reply:
Also caniuse is more focused on individual features than browsers, and [the feature](https://caniuse.com/#feat=css-crisp-edges) support is quite complicated, and I think my hack only works on Edge, or at least whatever supports an obscure canvas feature that has questionable support. This is mostly a user agent thing. I suppose I could make the js show up conditionally on the server side, instead of doing `window.navigator.userAgent`, but I don't think that's really worth the effort.